addon/services/cookies.js

```diff
@@ -116,31 +114,34 @@ export default Ember.Service.extend({
   _filterCachedFastBootCookies(fastBootCookiesCache) {
     let { hostname, path: requestPath, protocol } = this.get('_fastBoot.request');
 
-    return _collection.reduce(fastBootCookiesCache, (acc, cookie, name) => {
```

Comment:
`return Ember.A(_collection).reduce(…` doesn't work because we're iterating over `fastBootCookiesCache`, not `_collection`.  `return Ember.A(fastBootCookiesCache).reduce(…` however also doesn't work because `fastBootCookiesCache` is an object, not an array. it seems that the only reason for using lodash here was that lodash can iterate over the key-value pairs of an object directly, but I don't think that's worth the big dependency if a simple for-in loop can do the same.